apps/asctb-reporter/src/app/components/home/home.component.html

```diff
@@ -69,6 +80,8 @@
             mat-icon-button
             [hraIconButtonVariant]="'color'"
             [hraIconButtonSize]="'large'"
+            hraFeature="email"
+            [hraClickEvent]="{ contributor: contributor.name }"
```

Comment:
Add a `[hraFeature]="contributor.name | slugify"` on the hra-profile-card instead of passing this as a property each time